bin/helpers/load_run_environment.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+if [ -f .env ]; then
+   source .env
+
+   [ -n "$MYSTERIUM_API_URL" ] && DISCOVERY="--discovery-address=$MYSTERIUM_API_URL"
+   [ -n "$NATS_SERVER_IP" ] && BROKER="--broker-address=$NATS_SERVER_IP"
```

Comment:
`DISCOVERY` and `BROKER` are too common - perhaps `DISCOVERY_OPTION` and `BROKER_OPTION` are more explicit?